cmd/buildah/commit.go

```diff
@@ -51,6 +51,14 @@ var (
 			Name:  "signature-policy",
 			Usage: "`pathname` of signature policy file (not usually used)",
 		},
+		cli.StringFlag{
```

Comment:
Should we just make these global options of the buildah command rather then adding them to a bunch of commands.

Reply:
Hmm, no harm in that, it'd just be ignored a lot of the time.